ranger/data/scope.sh

```diff
@@ -75,20 +75,52 @@ handle_extension() {
         odt|ods|odp|sxw)
             # Preview as text conversion
             odt2txt "${FILE_PATH}" && exit 5
+	    # Preview as markdown conversion
+	    pandoc -s -t markdown -- "${FILE_PATH}" && exit 5
             exit 1;;
 
+	# RTF and DOC
+	rtf|doc)
+	    # Preview as text conversion
+	    # Note: catdoc does not always work for .doc files
+	    catdoc -- "${FILE_PATH}" && exit 5
+	    exit 1;;
+
+	# DOCX, ePub, FB2 (using markdown)
+	# You might want to remove "|epub" and/or "|fb2" below if you have uncommented other methods to preview those formats
```

Comment:
Maybe you should add a pandoc line to the section for epub and fb2 instead? Like you did for html?